packages/transformers/mdx/src/MDXTransformer.js

```diff
@@ -1,19 +1,18 @@
 // @flow
 import {Transformer} from '@parcel/plugin';
-import mdx from '@mdx-js/mdx';
 
 export default (new Transformer({
   async transform({asset}) {
+    let { compile } = await import('@mdx-js/mdx');
     let code = await asset.getCode();
-    let compiled = await mdx(code);
 
-    asset.type = 'js';
-    asset.setCode(`/* @jsxRuntime classic */
-/* @jsx mdx */
-import React from 'react';
-import { mdx } from '@mdx-js/react'
-${compiled}
-`);
+    try {
+      let compiled = await compile(code);
```

Comment:
This plugin should have the possibility to have options, like the webpack loader: https://mdxjs.com/packages/loader/#options

So that is people what to pass to parcel-mdx `remarkPlugins` or others, they'll have the possibility to

Reply:
as similar system as done in the Stylus plugin could be used: https://github.com/parcel-bundler/parcel/blob/a6765b2/packages/transformers/stylus/src/StylusTransformer.js#L18-L26

We try to load a `.mdxrc.js` or `.mdxrc.cjs` or `.mdxrc.mjs` file and if found, we disable the config on load with `config.invalidateOnStartup();` and this config is then passed to the mdx compiler as a 2nd argument.